src/main/java/wooteco/subway/line/controller/LineController.java

```diff
@@ -0,0 +1,59 @@
+package wooteco.subway.line.controller;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import wooteco.subway.line.dto.LineRequest;
+import wooteco.subway.line.dto.LineResponse;
+import wooteco.subway.line.service.LineService;
+
+import java.net.URI;
+import java.util.List;
+
+@RestController
+public class LineController {
+    private static final Logger log = LoggerFactory.getLogger("console");
+
+    private final LineService lineService;
+
+    public LineController(LineService lineService) {
+        this.lineService = lineService;
+    }
+
+    @PostMapping("/lines")
+    public ResponseEntity<LineResponse> createLine(@RequestBody LineRequest lineRequest) {
+        LineResponse newLine = lineService.save(lineRequest);
+        log.info("An INFO Message : {}", newLine.getName() + " 노선 생성 성공");
+        return ResponseEntity.created(URI.create("/lines/" + newLine.getId())).body(newLine);
+    }
+
+    @GetMapping(value = "/lines", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<List<LineResponse>> showLines() {
+        List<LineResponse> allLines = lineService.findAllLines();
+        log.info("An INFO Message : {}", "지하철 모든 노선 조회 성공");
+        return ResponseEntity.ok().body(allLines);
+    }
+
+    @GetMapping(value = "/lines/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<LineResponse> showLine(@PathVariable Long id) {
+        LineResponse line = lineService.findLine(id);
+        log.info("An INFO Message : {}", line.getName() + "노선 조회 성공");
+        return ResponseEntity.ok().body(line);
+    }
+
+    @PutMapping(value = "/lines/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<Void> updateLine(@PathVariable Long id, @RequestBody LineRequest lineRequest) {
+        lineService.updateLine(id, lineRequest);
```

Comment:
```suggestion
        lineService.update(id, lineRequest);
```
라인 서비스니까 당연히 라인을 업데이트할 것으로 기대하겠죠?
다른 객체를 업데이트 한다면 메서드명에 영향이 가는 객체명을 써주는게 의미가 있겠네요.